modules/apps/portal-template/portal-template-react-renderer-impl/npmscripts.config.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	federation: true,
+	preset: '@liferay/npm-scripts/src/presets/standard',
```

Comment:
Do we need the preset now? 🤔

Reply:
I was just wondering if we merged this by default or not 🤔

Reply:
Finally, don't we need a `runLegacyBuild: false` here? Does this mean we're still doing the bundler build and on top the webpack one? Who does what?

Reply:
For the sames reasons I commented in the previous comment, yes. As soon as you create a `npmscripts.config.js` file you must put the preset inside it or, otherwise, it is not used.

Reply:
`runLegacyBuild` inhibits `babel` and `bundler` so, unless we do everything through webpack we need them (in the intermediate stage). I will check if we still need it here.